src/_scss/pages/dashboard/_typeaheadFilter.scss

```diff
@@ -39,4 +39,57 @@
             }
         }
     }
-}
\ No newline at end of file
+
+  .filterdropdown {
+    .btn-default {
+      color: $color-white;
+      background-color: transparent;
```

Comment:
Can you add a fixed width so that the width of the button doesn't change when clicked (the font becomes bold)? (same for the Created By button)